src/ferc_eia_match/inputs.py

```diff
@@ -66,9 +67,16 @@ def __init__(
                 and all plant parts will be included in the EIA data.
         """
         self.pudl_out = pudl_out
+        self.pudl_out.freq = "AS"
```

Comment:
Forcibly assigning the `pudl_out` frequency could result in unexpected behavior. What if someone passes in an object with `freq=="MS"` that has already cached some EIA data at monthly frequency? I'd probably

```python
assert pudl_out.freq == "AS"
```